Resources/Prototypes/Entities/Mobs/base.yml

```diff
@@ -46,6 +46,14 @@
   - type: LanguageSpeaker # Einstein Engines. This component is required to support speech, although it does not define known languages.
   - type: RequireProjectileTarget
     active: False
+  - type: AnimatedEmotes
+
+- type: entity
+  save: false
+  id: MobPolymorphable
+  abstract: true
+  components:
+  - type: Polymorphable
   - type: OwnInteractionVerbs
     allowedVerbs: [] # TODO: define something here, or don't.
```

Comment:
This should belong to BaseMob as it always did.

Reply:
That is the point, yes.

Reply:
That is not the point, you nuked interaction verbs on all mobs by doing that

Reply:
That's crazy